presigned-pause.just

```diff
@@ -156,3 +156,66 @@ simulate-all:
       --json-file ${taskPath}/tx/draft-${nonce}.json \
       simulate
   done
+
+upload-PSPs:
+ #!/bin/bash
+ # Navigate to the directory containing the JSON files
+ # Uncomment and modify the next line if you need to change directories
+ echo "=================== PSPs Uploader ============================="
+ echo "This is about to Upload the PSPs (ready-X.sh.b64 and ready-X.json) to 1Password"
+ echo "Please make sure you have the files in the tx/ folder, if the PSPs is already uploaded this will duplicates it."
+ # Check if there is one-password cli installed on the system.
+ if which op >/dev/null 2>&1; then
+    echo "op is installed."
```

Comment:
nit, because I'm sure myself and others will forge that op here is 1password and not related to OP
```suggestion
    echo "op (one-password cli) is installed."
```